commitlint.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  extends: ["@commitlint/config-conventional"],
+  helpUrl:
+    "(example:- feat: testing lint) https://www.conventionalcommits.org/en/v1.0.0/#summary ",
```

Comment:
Can you change the example to something like this
(example: feat: add home page) [https://](https://www.conventionalcommits.org/en/v1.0.0/#summary)

What to change
- remove the `-` (nose) from the :- (nose and balls)
- no continuous/past form. Write in imperative (test, add, implement)

Rest is fine